PubSub/src/PubSubClient.php

```diff
@@ -17,19 +17,21 @@
 
 namespace Google\Cloud\PubSub;
 
-use Google\Auth\FetchAuthTokenInterface;
-use Google\Cloud\Core\ArrayTrait;
-use Google\Cloud\Core\ClientTrait;
+use Google\ApiCore\Traits\ArrayTrait;
+use Google\ApiCore\Traits\ClientTrait;
 use Google\Cloud\Core\Duration;
 use Google\Cloud\Core\Exception\BadRequestException;
-use Google\Cloud\Core\Iterator\ItemIterator;
-use Google\Cloud\Core\Iterator\PageIterator;
+use Google\ApiCore\Veneer\Iterator\ItemIterator;
```

Comment:
We should not expose this terminology to users. I think this was mentioned in a previous review, too.

Reply:
Sorry, comment-highlight is too broad — I'm referring to "Veneer" specifically.

Reply:
Yeah, I have already closed the gax PR that the class was moved to. We won't be using that anymore.